Contributing.md

```diff
@@ -1,7 +1,3 @@
-### Sign the CLA
-
-All contributors to your PR must sign our [Individual Contributor License Agreement (CLA)](https://spreadsheets.google.com/spreadsheet/viewform?formkey=dDViT2xzUHAwRkI3X3k5Z0lQM091OGc6MQ&ndplr=1). The CLA is a short form that ensures that you are eligible to contribute.
-
 ### Issues before features
```

Comment:
this file brings me back. reasonably sure I wrote an early draft of this bad boy.